test/utils/status.ts

```diff
@@ -0,0 +1,15 @@
+import nock from "nock";
+import { APP_CHECK_NAME } from "../../src/config";
+
+export const checkSuccessStatus = function (): void {
+  nock("https://api.github.com")
+    .post("/repos/tianhaoz95/approveman-test/check-runs", (body: any) => {
+      expect(body).toMatchObject({
+        name: APP_CHECK_NAME,
+        status: "completed",
+        conclusion: "success",
+      });
+      return true;
+    })
+    .reply(201);
```

Comment:
Codacy found an issue: ['magic numbers' are not allowed: 201](https://app.codacy.com/manual/tianhaoz/approveman/pullRequest?prid=6083797)